scalafix-tests/input/src/main/scala/test/organizeImports/ExplodeImportsFormatPreserving.scala

```diff
@@ -7,6 +7,6 @@ OrganizeImports.groupedImports = Explode
 package test.organizeImports
 
 import test.organizeImports.ExplodeImports.FormatPreserving.g1.{ a, b }
-import test.organizeImports.ExplodeImports.FormatPreserving.g2.{ c => C, _ }
+import test.organizeImports.ExplodeImports.FormatPreserving.g2.{ c  =>  C, _ }
```

Comment:
add whitespaces on purpose to see how rewriting with `as` behaves when the patch does not have the same length as what's being replaced